x-pack/plugins/enterprise_search/public/applications/applications/components/engine/engine_indices.tsx

```diff
@@ -174,22 +176,27 @@ export const EngineIndices: React.FC = () => {
   ];
   return (
     <>
-      {hasUnknownIndices && (
+      {(hasAllUnreachableIndices || hasUnknownIndices) && (
         <>
           <EuiCallOut
             color="warning"
             iconType="warning"
             title={i18n.translate(
               'xpack.enterpriseSearch.content.engine.indices.unknownIndicesCallout.title',
-              { defaultMessage: 'Some of your indices are unavailable.' }
+              {
+                defaultMessage: hasAllUnreachableIndices
+                  ? 'All of your indices are unavailable.'
+                  : 'Some of your indices are unavailable.',
+              }
```

Comment:
i think you'll have to make these separate calls to `i18n.translate`. translated strings should be statically defined.

Reply:
Replaced using FormattedMessage : 1e4ca774cd2a17d93bb13074f96ff26f5732e676